environment.yml

```diff
@@ -16,19 +16,19 @@ dependencies:
     - conda-forge::r-upsetr=1.3.3
 
     ## bioconda packages
-    - fastqc=0.11.8
-    - trim-galore=0.5.0
-    - bwa=0.7.17
-    - samtools=1.9
-    - picard=2.18.26
-    - bamtools=2.5.1
-    - pysam=0.15.2
-    - bedtools=2.27.1
-    - ucsc-bedgraphtobigwig=377
-    - macs2=2.1.2
-    - homer=4.9.1
-    - ataqv=1.0.0
-    - subread=1.6.3
-    - multiqc=1.7
+    - bioconda::fastqc=0.11.8
```

Comment:
Although I claimed that this speeds things up, according to https://github.com/nf-core/methylseq/issues/74#issuecomment-467974219 this is not the case..

Reply:
Ok. Ill change it back.